burn-import/tests/data/conv2d/conv2d.rs

```diff
@@ -0,0 +1,33 @@
+// Generated by integration tests
+use burn::nn::conv::Conv2d;
+use burn::nn::conv::Conv2dConfig;
+use burn::nn::PaddingConfig2d;
+use burn::{
+    module::Module,
+    tensor::{backend::Backend, Tensor},
+};
+
+
+#[derive(Module, Debug)]
+pub struct Model<B: Backend> {
+    conv2d1: Conv2d<B>,
```

Comment:
I think this is because this is the first conv2d field in the model. If other convolution layers existed we would have `conv2d1`, `conv2d2`, etc.

Reply:
Yes, this is used to add sequencing to the attribute names. I like it better than prefixing with underscore like this: x_1, y_1, instead of x1, and y2.

Reply:
@antimora 
That would be perfect, using `_` clearly states that we are changing the context